src/types.ts

```diff
@@ -0,0 +1,8 @@
+/** Common types */
+export type Maybe<T> = T | undefined | null;
```

Comment:
This type isn't required since `getError` doesn't require `Maybe` anymore, see comment above.

Reply:
done